judgeops/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
@TilakMaddy could you create a readme inside `judgeops` that walks through the process, please?

Reply:
DONE !!!